src/callbacks_step/save_solution_dg.jl

```diff
@@ -111,6 +125,8 @@ function save_solution_file(u, time, dt, timestep,
         n_vars = size(data, 1)
     end
 
+    # TODO: In trixi2txt, the data file is read; including node_variables for every mesh.
+    # So, do we need node_variables here as well?
```

Comment:
Since we read out `node_variables` here and in Trixi2Vtk as well, we have to add (a possibly empty) `node_variables` for every mesh type. Is there missing one right now?

Reply:
> Is there missing one right now?

I don't understand the question, can you try to elaborate?

Reply:
It was more a reminder for me to check if `node_variables` are initialized for all mesh types.

And I think I did not implement it for the meshes `ParallelTreeMesh` and `ParallelP4estMesh`.
The function [`save_solution_file` for these meshes](https://github.com/bennibolm/Trixi.jl/blob/subcell-limiting-visualization/src/callbacks_step/save_solution_dg.jl#L93-L139) has the parameter but doesn't do anything with it. Therefore, I think this will generate errors. 

Moreover, I cannot find a `save_solution_file` routine for `DGMultiMesh`. This doesn't use the SaveSolution callback at all, does it?

Reply:
> Moreover, I cannot find a `save_solution_file` routine for `DGMultiMesh`. This doesn't use the SaveSolution callback at all, does it?

cc @jlchan ?

Reply:
I don't think so...I believe @Davknapp implemented some VTK writing, but I don't think we have a `SaveSolution` callback yet?

Reply:
When I had a first look into Trixi.jl i once started a branch on this work. But I dropped it long ago and didn't put any further work into it. Feel free to have a look at it and change as much as you want ;)